client/components/ShareButton.js

```diff
@@ -17,16 +19,34 @@ const useStyles = ((canShare) => makeStyles((theme) => ({
 })));
 
 export default () => {
+  const [state, setState] = useState({showSnackbar: false});
```

Comment:
I generally try to keep state out of components as much as possible and move it to redux state with appropriate reducers and sagas. Don't expect you to refactor this but I'll give it a shot so we can have an easier and more general way of showing toasts in the app :-) I've opened #27 for me and I'll take a look on the weekend.

Reply:
> I generally try to keep state out of components as much as possible

Hmm, you may not like [what I had planned for my next PR](https://github.com/glennhartmann/draw/commit/c098f772215f28310933ec127ce094d39e681b8e), then...

> and move it to redux state with appropriate reducers and sagas

No idea what any of this means, but go for it.